qa/L0_sequence_batcher/test.sh

```diff
@@ -827,8 +838,14 @@ fi
 
 ### Start Preserve Ordering Tests ###
 
-# Test only supported on windows currently due to use of python backend models
-if [ ${WINDOWS} -ne 1 ]; then
+# FIXME: Test only supported on windows currently due to use of python backend models.
+# Now that Windows supports the PYBE, we should check that this tests works once Windows
+# CI is stable.
+
+# These subtests use python models. They should not be executed if 'python' is not one
+# of the backends under test.
+should_test_backend "python"
+if [ $? -eq 1 ] && [ ${WINDOWS} -ne 1 ]; then
```

Comment:
is this variable something we must introduce manually

Reply:
If you mean manually provide the arg list -- yes.

The context here is that the enclosed test below runs tests that use Python models. Traditionally, our tests are written such that we assume that all backends are supported and exclude certain tests based on the target platform (e.g., Windows, Jetson, etc.). This didn't strike me as the right approach. Instead, my thought was to leverage the `BACKENDS` variable, which should generally indicate which backends are supported, to include/exclude certain tests.

This was just an idea I was playing with for this PR, but I'm open to feedback. Alternatively, we can go back to our traditional approach and introduce an external $RHEL_TEST flag or something like we do for Windows here.